src/03-modality-agnostic-files.md

```diff
@@ -81,29 +85,105 @@ Template:
 ```Text
 participants.tsv
 participants.json
-phenotype/<measurement_tool_name>.tsv
-phenotype/<measurement_tool_name>.json
 ```
 
-Optional: Yes
+The purpose of this RECOMMENDED file is to describe properties of participants
+such as age, sex, handedness etc.
+In case of single session studies this file has one compulsory column
+`participant_id` that consists of `sub-<label>`, followed by a list of optional
+columns describing participants.
+Each participant MUST be described by one and only one row.
+
+Commonly used *optional* columns in `participant.tsv` files are `age`, `sex`,
+and `handedness`. We RECOMMEND to use the following values for these three
+columns in case that you use them:
+
+-   `age`: numeric value in years (float or integer value)
+
+-   `sex`: string value indicating phenotypical sex, one of "male", "female",
+    "other"
+
+    -   for "male", use one of these values: `male`, `m`, `M`, `MALE`, `Male`
 
-The purpose of this file is to describe properties of participants such as age,
-handedness, sex, etc. In case of single session studies this file has one
-compulsory column `participant_id` that consists of `sub-<label>`,
-followed by a list of optional columns describing participants. Each participant
-needs to be described by one and only one row.
+    -   for "female", use one of these values: `female`, `f`, `F`, `FEMALE`,
```

Comment:
do we need to use the keyword MUST here?

Reply:
see some lines above `We RECOMMEND to use the following values`